lib/ConvoyLeader/src/App.cpp

```diff
@@ -215,33 +216,44 @@ void App::loop()
     /* Process Longitudinal Controller */
     m_longitudinalController.process();
 
-    if (false == m_initialDataSent)
+    if (true == m_initialCommandTimer.isTimeout())

```

Comment:
The whole block will later be part of an initial state, right?

Reply:
Yes!

Reply:
Ok.